specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2023-12-01-preview/examples/CreateSystemComponent.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "subId1",
+    "resourceGroupName": "rg1",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterResourceName": "connectedClusters",
+    "api-version": "2023-05-01-preview",
+    "clusterName": "clusterName1",
+    "systemComponents": {
+      "properties": {
+        "components": [
+          {
+            "extensionType": "Strato",
+            "userSpecifiedVersion": "0.1.4"
+          },
+          {
+            "extensionType": "SomeOtherSystemExtension",
+            "userSpecifiedVersion": "0.1.1"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "description": "Details of the Kubernetes SystemComponent's current status.",
+      "body": {
+        "id": "/subscriptions/subId1/resourceGroups/rg1/providers/Microsoft.Kubernetes/connectedClusters/clusterName1/providers/Microsoft.KubernetesConfiguration/systemComponents/default",
+        "type": "Microsoft.KubernetesConfiguration/systemComponents",
+        "name": "default",
+        "properties": {
+          "arcAgentryAutoUpgrade": false,
+          "status": "Succeeded",
+          "components": [
+            {
+              "extensionType": "Strato",
+              "userSpecifiedVersion": "0.1.4",
+              "CurrentInstalledVersion": "0.1.9",
+              "installedMajorVersion": "0",
+              "status": "Succeeded"
+            },
+            {
+              "extensionType": "SomeOtherSystemExtension",
+              "userSpecifiedVersion": "0.1.1",
+              "currentInstalledVersion": "0.1.1",
+              "installedMajorVersion": "0",
```

Comment:
It's kind of dumb but could you set one of these two to have a major version of `1` since this is "documentation" for customers and it may help them better understand the `installedMajorVersion` property.

Reply:
may we can remove this.